### DISCUSSION TITLE
user database error

### DISCUSSION THREAD

**Discussion by siva-devops73** [learndevopsonline/learndevopsonline#1828] [Q&A]
Hi Raghu, can you look below one
![Screenshot 2024-02-23 202846](https://github.com/learndevopsonline/learndevopsonline/assets/134507877/cff03038-9e4a-45ba-b08a-0203b31dd8eb)
logs of user component
![Uploading Screenshot 2024-02-23 202831.png…]()

**Comment by siva-devops73** [comment]
logs of user component
![Screenshot 2024-02-23 202831](https://github.com/learndevopsonline/learndevopsonline/assets/134507877/1d034bca-3937-4c4c-bd89-8ad28eedc84b)

**Comment by chaithanya1812** [comment]
can you paste the workstation & frontend ip-addresses

**Reply by siva-devops73** [reply]
workstation 44.192.123.179
frontend private : 10.10.0.179

**Reply by siva-devops73** [reply]
i think you need user id right?

**Reply by chaithanya1812** [reply]
can you paste frontend url

**Reply by siva-devops73** [reply]
www.devops73.store

**Comment by chaithanya1812** [comment]
Hi Raghu sir can you check once, It's getting 500 error

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/a25290f1-44d7-4d8f-8062-965856d3cf2b)

**Comment by siva-devops73** [comment]
Hi Chaithanya, can you ignore above, now its working
i push new tag with some changes and it's working fine,
